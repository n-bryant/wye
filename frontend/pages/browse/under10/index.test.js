import { Under10Page } from "./index";

describe("Under10Page", () => {
  const props = {
    classes: {
      root: "root"
    }
  };

  it("should render successfully", () => {
    expect(toJson(shallow(<Under10Page {...props} />))).toMatchSnapshot();
  });
});
