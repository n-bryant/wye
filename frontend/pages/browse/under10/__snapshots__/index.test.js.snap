// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Under10Page should render successfully 1`] = `
<div
  className="wye-Under10Page root"
>
  <WithWidth(WithStyles(GamesFilterWidget))
    initialFilters={
      Object {
        "filters": Object {
          "gameFilters": Object {
            "finalPrice_lte": 999,
            "freeToPlay": false,
          },
        },
        "orderBy": Array [
          "FINAL_PRICE",
          "USER_RATING",
        ],
        "sortOrder": "DESC",
      }
    }
    subtitle="Browse games available at a bargain"
    title="Games Under $10"
  />
</div>
`;
